ui/article-aem-renderer/src/main/java/org/cru/godtools/article/aem/activity/AemArticleActivity.kt

```diff
@@ -49,10 +51,15 @@ class AemArticleActivity : BaseSingleToolActivity(false) {
         loadFragmentIfNeeded()
     }
 
+    private fun postAnalyticScreen(title: String) {
+        mEventBus.post(AnalyticsScreenEvent(title, getDeviceLocale(this)))
+    }
+
     private fun onUpdateArticle(article: Article?) {
         this.article = article
         updateToolbarTitle()
         updateVisibilityState()
+        postAnalyticScreen(this.article!!.title)
```

Comment:
this could fire multiple times, we need to figure out how we actually want to handle triggering this delayed screen event.